### ISSUE TITLE
gsutils missing crcmod in 289.0.0

### ISSUE THREAD

**Issue by seadowg** [GoogleCloudPlatform/cloud-sdk-docker#200]
Running `gsutil version -l` in `google/cloud-sdk:289.0.0` shows the following:

```
compiled crcmod: false
```

This caused a `gsutil` copy to fail for us in our CI. We've reverted back to `273.0.0` which is working fine.

**Comment by thewoolleyman** [comment]
This actually happened as of version 289.0.0.

Looking at the output from `gsutil version -l` in the docker images below, I suspect that it's the upgrade to python from `3.7` to `3.8` which broke the precompiled extension, as of image version `289.0.0`.  See **`compiled crcmod:`** changes from **`True`** to **`False`**

**From 288.0.0 image:**
```
root@3deff88cb413:/# gsutil version -l
gsutil version: 4.49
checksum: 89aa91d37decba4899431c0415120312 (OK)
boto version: 2.49.0
python version: 3.7.3 (default, Dec 20 2019, 18:57:59) [GCC 8.3.0]
OS: Linux 4.19.76-linuxkit
multiprocessing available: True
using cloud sdk: True
pass cloud sdk credentials to gsutil: True
config path(s): No config found
gsutil path: /usr/lib/google-cloud-sdk/bin/gsutil
compiled crcmod: True
installed via package manager: False
editable install: False
```

**From 289.0.0 image:**
```
root@0eee17ffa285:/# gsutil version -l
gsutil version: 4.49
checksum: 89aa91d37decba4899431c0415120312 (OK)
boto version: 2.49.0
python version: 3.8.2 (default, Apr  1 2020, 15:52:55) [GCC 9.3.0]
OS: Linux 4.19.76-linuxkit
multiprocessing available: True
using cloud sdk: True
pass cloud sdk credentials to gsutil: True
config path(s): No config found
gsutil path: /usr/lib/google-cloud-sdk/bin/gsutil
compiled crcmod: False
installed via package manager: False
editable install: False
```

**Comment by seadowg** [comment]
@thewoolleyman Whooops that's actually a typo in the issue title. You'll see in the description I was trying with `289.0.0`. I'll fix the title!

P.S. Hello from an ex-Pivot 👋

**Comment by thewoolleyman** [comment]
So, this is fixed by just running `pip3 install -U crcmod` on the image.

Which seems to imply that this doesn't work in the `Dockerfile` [when it is initially done](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/e5abe1c/Dockerfile#L20), but works later.

One question I have is: Why is the image [inheriting from `buster`](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/e5abe1c/Dockerfile#L3) and then [adding an apt source for `sid`](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/e5abe1c/Dockerfile#L21)?

**Comment by willfaris** [comment]
Fixed with https://github.com/GoogleCloudPlatform/cloud-sdk-docker/commit/2e81e4a496b9aa8940157ce52212b5371864a82e

New images will be pushed to gcr.io and Dockerhub soon.

**Comment by dnsmichi** [comment]
@willfaris Thanks for fixing this so fast :-) Can you tag a new release please too, for those pinning the image to releases? Thanks!

**Comment by willfaris** [comment]
290.0.1 (a hotfix release) was just built.  :latest no longer has the issue.

**Comment by thewoolleyman** [comment]
@willfaris Just FYI, now that there is no `pip3` on the image, anyone depending on it being there will have failures.

E.g., we had been manually building it as a workaround for this issue, and that started blowing up on all our builds as soon as this image was released, until we removed it:

```
 $ pip3 install -U crcmod
 /bin/bash: line 144: pip3: command not found
```

**Comment by willfaris** [comment]
Yup, I removed pip3 from the image. Since this is an image for Cloud SDK I decided to go ahead and remove it as it was really there to install crcmod as far as I can tell from history. In general the image needs to be cleaned.  If people need pip3 they can install it themselves. Sorry about breaking your builds, I assume your usage of pip3 was related to https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/200#issuecomment-618549897
